### PR TITLE
CAD-992:  updates for the latest deployment

### PR DESCRIPTION
**Pull request by deepfire**

### CODE REVIEW

File: cardano-tx-generator/src/Cardano/Benchmarking/GeneratorTx.hs [hunk 1]
```diff
@@ -179,7 +183,11 @@ genesisBenchmarkRunner loggingLayer
   when (tps < 0.05) $
     left $ TooSmallTPSRate tps
 
-  let (benchTracer, connectTracer, submitTracer, lowLevelSubmitTracer) = createTracers loggingLayer
+  let ( benchTracer
+       , connectTracer
+       , submitMuxTracer
```
Comment (CodiePP) [inline cardano-tx-generator/src/Cardano/Benchmarking/GeneratorTx.hs]: alignment off

File: cardano-tx-generator/src/Cardano/Benchmarking/GeneratorTx.hs [hunk 2]
```diff
@@ -1022,16 +1046,17 @@ txGenerator benchTracer
             txsListsForTargetNodes = do
   liftIO . traceWith benchTracer . TraceBenchTxSubDebug
     $ " Prepare to generate, total number of transactions " ++ show numOfTransactions
+      ++ ", for " ++ show numOfTargetNodes ++ " peers"
 
   -- Generator is producing transactions and writes them in the list.
   -- Later sumbitter is reading and submitting these transactions.
-  txsForSubmission :: MSTM.TMVar IO [CC.UTxO.ATxAux ByteString] <- liftIO $ STM.newTMVarIO []
+  txsForSubmission :: [MSTM.TMVar IO [CC.UTxO.ATxAux ByteString]] <-
+    liftIO $ replicateM numOfTargetNodes $ STM.newTMVarIO []
```
Comment (CodiePP) [inline cardano-tx-generator/src/Cardano/Benchmarking/GeneratorTx.hs:1059]: so this is not only introducing "cooldown"?
doese this still match the comment?

File: cardano-tx-generator/src/Cardano/Benchmarking/GeneratorTx/Submission.hs [hunk 3]
```diff
@@ -390,34 +393,45 @@ data TraceBenchTxSubmit txid
...
-  toJSON txId = A.String (T.pack $ show txId)
+  toJSON txid = A.String (T.pack $ show txid)
 
 instance ToObject (TraceBenchTxSubmit (Mempool.GenTxId ByronBlock)) where
   toObject MinimalVerbosity _ = emptyObject -- do not log
   toObject NormalVerbosity t =
     case t of
       TraceBenchTxSubRecv _      -> mkObject ["kind" .= A.String "TraceBenchTxSubRecv"]
       TraceBenchTxSubStart _     -> mkObject ["kind" .= A.String "TraceBenchTxSubStart"]
+      TraceBenchTxSubServAnn _  -> mkObject ["kind" .= A.String "TraceBenchTxSubServAnn"]
```
Comment (CodiePP) [inline cardano-tx-generator/src/Cardano/Benchmarking/GeneratorTx/Submission.hs]: alignment off

File: benchmarks/cluster3nodes/configuration/parameters [hunk 4]
```diff
@@ -6,7 +6,7 @@ numtx=1000
 targetnodes="0 1 2"
```
Comment (CodiePP) [inline benchmarks/cluster3nodes/configuration/parameters:6]: I think this file should not be changed by the PR

**Review by CodiePP — CHANGES_REQUESTED** [review]
some comments; needs thorough testing..

File: cardano-tx-generator/src/Cardano/Benchmarking/GeneratorTx.hs (hunk 1 above)
Comment (deepfire) [reply cardano-tx-generator/src/Cardano/Benchmarking/GeneratorTx.hs]: It's not off -- it's a parse error if I align `,` with `(`.

File: cardano-tx-generator/src/Cardano/Benchmarking/GeneratorTx.hs (hunk 2 above)
Comment (deepfire) [reply cardano-tx-generator/src/Cardano/Benchmarking/GeneratorTx.hs:1059]: This is a part of the fix for the single-threaded submission bug.

..and yes, I think the comment is still true.

**Comment by deepfire** [comment]
@CodiePP, addressed all comments.

File: benchmarks/cluster3nodes/configuration/parameters (hunk 4 above)
Comment (deepfire) [reply benchmarks/cluster3nodes/configuration/parameters:6]: Done.

File: cardano-tx-generator/src/Cardano/Benchmarking/GeneratorTx/Submission.hs (hunk 3 above)
Comment (deepfire) [reply cardano-tx-generator/src/Cardano/Benchmarking/GeneratorTx/Submission.hs]: Done.

File: cardano-tx-generator/src/Cardano/Benchmarking/GeneratorTx.hs (hunk 1 above)
Comment (CodiePP) [reply cardano-tx-generator/src/Cardano/Benchmarking/GeneratorTx.hs]: then it might be better to put the comma at the end of the line and align the names?

**Comment by deepfire** [comment]
Closing in favor or componentised PRs.

**Pull request closed without merging**